source/SkiaSharp.Views/SkiaSharp.Views.WPF/SkiaSharp.Views.WPF.csproj

```diff
@@ -1,12 +1,13 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
   <PropertyGroup>
-    <TargetFramework>net45</TargetFramework>
+    <TargetFrameworks>net451;netcoreapp3.0</TargetFrameworks>
```

Comment:
I see you changed this to 451, is there a reason?

Either way, the build is failin on the packaging because the nuspec still expects net45:
> Could not find a part of the path 'D:\a\1\s\output\SkiaSharp.Views.WPF\nuget\lib\net45'.

Reply:
Oh, it appears not.  Sorry about that.

I would have bet money that I saw some other `net45` to `net451` change between the `master` and `dev/vs-2019` branches...and then I would have lost that money!

I just pushed a commit that reverts this back to `net45`.